app/Http/Controllers/QueryController.php

```diff
@@ -159,9 +168,80 @@ public function membershipTotals(Request $request)
+        $count_honorary = Member::query()->where('membership_type', MembershipTypeEnum::HONORARY)->count();
+        $count_donor = Member::query()->where('membership_type', MembershipTypeEnum::DONOR)->count();
+        $count_pending = Member::query()->where('membership_type', MembershipTypeEnum::PENDING)->count();
+        $count_pet = Member::query()->where('membership_type', MembershipTypeEnum::PET)->count();
+
+        $count_primary = Member::query()->where('membership_type', MembershipTypeEnum::REGULAR)->where('is_primary_at_another_association', false)->whereHas('UtAccount')->count();
+        $count_secondary = $count_total - $count_primary;
+        $count_ut = Member::query()->where('membership_type', MembershipTypeEnum::REGULAR)->where(function ($query) {
+            $query->whereHas('UtAccount')->orWhereHas('user', function ($query) {
+                $query->whereNotNull('utwente_username');
+            });
+        })->count();
+
+        $membersWhoArentPrimaryAnymore = Member::query()->whereDoesntHave('UtAccount')->where('primary', true)->get();
+
+        $membersWhoAreNewPrimary = Member::query()->whereHas('UtAccount')->where('primary', false)->get();
+
+        return view('queries.membership_totals', [
+            'total' => $count_total,
+            'primary' => $count_primary,
+            'secondary' => $count_secondary,
+            'ut' => $count_ut,
+            'active' => $count_active,
+            'lifelong' => $count_lifelong,
+            'honorary' => $count_honorary,
+            'donor' => $count_donor,
+            'pending' => $count_pending,
+            'pet' => $count_pet,
+            'membersWhoArentPrimaryAnymore' => $membersWhoArentPrimaryAnymore,
+            'membersWhoAreNewPrimary' => $membersWhoAreNewPrimary,
         ]);
     }
 
+    public function primaryExport()
+    {
+        $export_subsidies = [];
+        /** @var User[] $users */
+        $users = User::query()->whereHas('member', function ($query) {
+            $query->where('membership_type', MembershipTypeEnum::REGULAR)->whereHas('UtAccount', function ($q) {
+                $q->where('is_primary_at_another_association', false);
+            });
+        })->with('member.UtAccount')->get();
+        foreach ($users as $user) {
+            $export_subsidies[] = (object) [
+                'name' => $user->name,
+                'primary' => 'true',
+                'email' => $user->member->UtAccount()->first()->mail,
+                'ut_number' => $user->member->UtAccount()->first()->number,
+            ];
+        }
+
+        $headers = [
+            'Content-Encoding' => 'UTF-8',
+            'Content-Type' => 'text/csv; charset=UTF-8',
+            'Content-Disposition' => sprintf('attachment; filename="primary_member_overview_%s.csv"', date('d_m_Y')),
+        ];
+
+        return Response::make(view('queries.export_subsidies', ['export' => $export_subsidies]), 200, $headers);
```

Comment:
I'm not seeing the view

Reply:
The view is there? It is already an existing view